terraform/performance/amis.tf

```diff
@@ -0,0 +1,42 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+variable "ami_family" {
```

Comment:
The `ami_family` will use as a holder variable that respond based on the OS (if the family is linux, the ami_family will return a map of variables that will be used for linux - install_package with amazon-cloudwatch-agent.rpm and ec2-user if in linux). In this way, we will reduce the amount of variables we need to specify through terraform (e.g only terraform apply without terraform apply --user="ec2-user" if in linux)